_includes/install.md

```diff
@@ -1,17 +1,23 @@
+#### Requirements
+
 Currently, the Effekt language is implemented in Scala and compiles to JavaScript (besides other backends).
 To use the Effekt compiler and run the resulting programs, you need to have
 
-- Java (>= 11),
-- [Node.js](https://nodejs.org/en/) (>= 12), and
+- Java (>= 17),
```

Comment:
Why? :)
We're still supporting Java 11, aren't we?

Reply:
Yes, you are right

https://github.com/effekt-lang/effekt/blob/master/.github/workflows/ci.yml#L10

I thought we updated that even in the CI.